internal/venice-common/src/main/java/com/linkedin/venice/kafka/partitionoffset/PartitionOffsetFetcherImpl.java

```diff
@@ -48,15 +48,10 @@ public class PartitionOffsetFetcherImpl implements PartitionOffsetFetcher {
 
   private final Logger logger;
   private final Lock adminConsumerLock;
-  private final Lock kafkaAdminWrapperLock;
   private final Lazy<PubSubAdminAdapter> kafkaAdminWrapper;
   private final Lazy<PubSubConsumerAdapter> pubSubConsumer;
   private final Duration kafkaOperationTimeout;
 
-  private enum TopicEndType {
```

Comment:
What is this? Is it missing a rebase?

Reply:
It is from previous change, I just find it is unused in this class so I removed it.

Reply:
Got it, SG.